Add influxdb3 query parquet output example (#6735)
influxdata/docs-v2: content/shared/influxdb3-admin/databases/list.md

{{% /expand %}}
{{< /expand-wrapper >}}

### Output to a Parquet file

> You can also use the `influxdb3 query` command to output your list of databases to a Parquet file

Reviewer: ```suggestion
To output your list of databases to a Parquet file, use the `influxdb3 query` command 
```

  --header "Authorization: Bearer AUTH_TOKEN"
```

> Parquet responses contain the same database list, but you must provide an output destination because the format is binary.

Reviewer: ```suggestion
The response contains the databases list.

For Parquet responses, you must provide an output destination because the format is binary.
```
